src/css-validator/index.js

```diff
@@ -0,0 +1,105 @@
+var _ = require('underscore');
+var path = require('path');
+var StyleStats = require('stylestats');
+var TooManyRulesError = require('../errors/too-many-rules-error');
+var FileTooBigError = require('../errors/file-too-big-error');
+
+var MAX_CSS_RULES = 4095;
+var MAX_FILE_SIZE_KB = 278;
+
+var cssStatsSettings = {
+    published: false,
+    paths: false,
+    stylesheets: false,
+    styleElements: false,
+    size: true,
+    dataUriSize: false,
+    ratioOfDataUriSize: false,
+    gzippedSize: false,
+    simplicity: false,
+    rules: true,
+    selectors: false,
+    declarations: false,
+    averageOfIdentifier: false,
+    mostIdentifier: false,
+    mostIdentifierSelector: false,
+    averageOfCohesion: false,
+    lowestCohesion: false,
+    lowestCohesionSelector: false,
+    totalUniqueFontSizes: false,
+    uniqueFontSizes: false,
+    totalUniqueFontFamilies: false,
+    uniqueFontFamilies: false,
+    totalUniqueColors: false,
+    uniqueColors: false,
+    idSelectors: false,
+    universalSelectors: false,
+    unqualifiedAttributeSelectors: false,
+    userSpecifiedSelectors: false,
+    importantKeywords: false,
+    floatProperties: false,
+    mediaQueries: false
+};
+
+var whitelistedBundleNames = ['mobile', 'csr', 'pulse'];
+
+var shouldValidate = function(bundle) {

```

Comment:
We don't bother validating if it's a mobile or CSR/Pulse bundle since we don't have to worry about IE limits.